Source/Scene/ImplicitTileset.js

```diff
@@ -139,3 +191,49 @@ export default function ImplicitTileset(baseResource, tileJson) {
    */
   this.maximumLevel = extension.maximumLevel;
 }
+
+/**
+ * Gather JSON headers for all contents in the tile.
+ * This handles both regular tiles and tiles with the
+ * `3DTILES_multiple_contents` extension
+ *
+ * @param {Object} tileJson The JSON header of the tile with the 3DTILES_implicit_tiling extension.
+ * @return {Object[]} An array of JSON headers for the contents of each tile
+ * @private
+ */
+function gatherContentHeaders(tileJson) {
+  if (has3DTilesExtension(tileJson, "3DTILES_multiple_contents")) {
+    return tileJson.extensions["3DTILES_multiple_contents"].content;
```

Comment:
I wonder, in the spec should we enforce that `content` length is greater than or equal to two?

https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/extensions/3DTILES_multiple_contents/0.0.0/schema/tile.3DTILES_multiple_contents.schema.json#L13

Reply:
I'm on the fence about this one. While yes, there's really no point in using multiple contents with a single content, it doesn't hurt anything. Either way you still need to check if the tile is using single or multiple contents since they're stored in two different places in the JSON.

The fact that implicit tiling will transcode a single multiple-content to a regular tile is just an implementation detail (an optimization even) as far as I'm concerned.

Reply:
After thinking about this more, I tend to agree with you